syntax.d/xml.ini

```diff
@@ -0,0 +1,9 @@
+name=XML
+extensions=xml
+highlight_numbers=true
+singleline_string_quotes="
+singleline_comment_start=<!--
```

Comment:
I think this line should be removed, since <!-- starts a multi-line comment.
This way, if you start typing a multi-line comment, the entire comment (not just the first line) is highlighted right away